applications/nrf_desktop/README.rst

```diff
@@ -331,6 +331,44 @@ Add the ``pm_static_${CMAKE_BUILD_TYPE}.yml`` partition manager configuration fi
 
 For more information about how to configure the flash memory layout using the partition manager, see :ref:`partition_manager`.
 
+Device Firmware Upgrade
+***********************
+
+The nRF Desktop application uses background image transfer for the DFU process.
+The following elements take part in the firmware update process:
+
+    * `Update image generation`_
+    * `Update image transfer`_
+    * `Update image verification and swap`_
+
+Update image generation
+=======================
+
+The update image is automatically generated if the bootloader is enabled and can be found at the application build directory.
```

Comment:
I will add more information here under subsequent PR when the switch to B0 is complete.

Reply:
Done